android/src/main/java/com/applovin/reactnative/AppLovinMAXModule.java

```diff
@@ -401,51 +403,51 @@ public void onDismiss()
         } );
     }
 
-    @ReactMethod(isBlockingSynchronousMethod = true)
-    public int getConsentDialogState()
+    @ReactMethod
+    public void getConsentDialogState(final Promise promise)
```

Comment:
We recently [deprecated this API](https://app.asana.com/0/573104092700345/1202849413156202). Could you create a separate PR to deprecate it so developers get a warning, then another PR to be merged in the release after actually removing the API?

Reply:
Is 'showConsentDialog()' also deprecated?   I'm looking at [this](https://github.com/AppLovin/Unity-Plugin/pull/402) for reference.

Reply:
I just noticed that Android studio didn't indicate that showConsentDialog is deprecated, so it is not deprecated.